### DISCUSSION TITLE
Wetland Assessment Project: Static Text Review

### DISCUSSION THREAD

**Discussion by marshallrobinson** [UGS-GIO/wetlands#38] [Q&A] [answered]
Hi @wetlandsDiane, I'm working through the review of the Wetlands app (Jay's most recent version) and have a couple questions:

1) The 'Wetland Assessment Project' popups have new static text in them, and I'm unsure the review it has gone through. Has Steph C. reviewed it yet? 

2) There are 'Project Tables' in the 'Wetland Assessment Study Results' layer with new text that I'm unsure of their review as well. Has Steph C reviewed it yet? I'm only concerned about the text that is in sentence-format.

The more I review the application, I realize that the text in the 'Project Tables' is the same that shows up in the popups. Sorry about the redundant questions.

**Comment by wetlandsDiane** [comment, marked as answer]
I just looked through a bunch of emails and didn't see any evidence that
Stephanie reviewed the text. So unless there is a note in GitHub, I would
guess she hasn't reviewed it yet.

On Tue, Mar 8, 2022 at 5:00 PM marshallrobinson ***@***.***>
wrote:

> Hi @wetlandsDiane <https://github.com/wetlandsDiane>, I'm working through
> the review of the Wetlands app (Jay's most recent version) and had a
> question:
>
> The 'Wetland Assessment Project' popups have new static text in them, and
> I'm unsure the review it has gone through. Has Steph C. reviewed it yet?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/UGS-GIO/wetlands/discussions/38>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE3KRTS4TSSYQMI5RR56QGTU67S2JANCNFSM5QH7KNNA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Diane Menuz
Wetland Program Coordinator
Utah Geological Survey
1594 W. North Temple
P.O. Box 146100
Salt Lake City, UT 84114
801.538.7436
http://geology.utah.gov/resources/wetlands/

**Comment by marthajensen** [comment]
@geowomansteph   

Do you mind reviewing the text in the attached CSV for the wetlands app? The target population, target population comparison, and the sample frame fields all have full-sentence text that shows in the new app. Let me know what changes need to be made to the text. 
Thx! Martha

[WetlandAppText.csv](https://github.com/UGS-GIO/wetlands/files/8216321/WetlandAppText.csv)

**Comment by geowomansteph** [comment]
Sorry about that....I flaked. Here's the reviewed/edited cvs file
[WetlandAppText.csv](https://github.com/UGS-GIO/wetlands/files/8216617/WetlandAppText.csv)
.

**Comment by marthajensen** [comment]
Thx @geowomansteph 

The data in the ArcGIS Server service has been updated with Stephanie's reviewed text
